File: starter-packs/python/src/bot.py
import random
from core.action import Action, Direction
from core.game_state import GameState

class MyBot:
    def __init__(self):
        self.__name = "MyBot"

    def tick(self, state: GameState) -> Action:
       print(state.__str__())
       # program your bot here
       

       return random.choice(list(Direction))

File: starter-packs/javascript/src/bot.js
import { Action, Direction, Pattern } from '../core/action.js';

class MyBot {
  constructor() {
    this.name = "test ";
    this.first_turn = true;
  }

  tick(state) {
    if (this.first_turn) {
      this.first_turn = false;
      return new Action(new Pattern([Direction.UP, Direction.RIGHT]));
    }

    const directions = [Direction.UP, Direction.DOWN, Direction.LEFT, Direction.RIGHT];

    return new Action(directions[Math.floor(Math.random() * directions.length)]);
  }
};

export { MyBot };